code/modules/vending/wardrobes.dm

```diff
@@ -70,6 +79,10 @@
 					/obj/item/clothing/suit/hazardvest = 3,
 					/obj/item/clothing/shoes/workboots = 3,
 					/obj/item/clothing/head/hardhat = 3)
+	contraband = list(/obj/item/storage/toolbox/mechanical = 1,
+					  /obj/item/storage/toolbox/electrical = 2,
+					  /obj/item/storage/toolbox/artistic = 1)
+	premium = list(/obj/item/storage/box/stockparts/basic = 1)
```

Comment:
No please. The toolboxes should be Premium, considering they have tools already inside of them. The stock parts box shouldn't even be on this list. THIS IS A WARDROBE. You should put the toolboxes in the toolmate.